src/modules/functionsServer.ts

```diff
@@ -27,22 +25,63 @@ export const functionsServer = async (basePath: string, port: number) => {
     })
   )
 
-  console.log('')
-
   app.all('/*', async (req: Request, res: Response) => {
     const [endpoint] = req.params['0'].split('/')
 
-    const lambdaEvent = {
+    const queryStringParameters: { [key: string]: string } = {}
+
+    Object.entries(req.query).forEach(([key, value]) => {
+      if (value !== undefined) {
+        queryStringParameters[key] = value.toString()
+      }
+    })
+
+    const headers: { [key: string]: string } = {}
+
+    Object.entries(req.headers).forEach(([key, value]) => {
+      if (value !== undefined) {
+        if (Array.isArray(value)) {
+          headers[key] = value[value.length - 1]
```

Comment:
Why when it is an array you forward only the last element of the array?

Reply:
Good one!

I thought HTTP/1.1 couldn't have multiple headers with the same name, but [reading the specs](https://tools.ietf.org/html/rfc2616#section-4.2) I see that it's allowed in some circumstances. I will join them separated by comma, since lambda headers are defined as `{[key: string]: string}` as per the [specs](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html) and [implementation](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/aws-lambda/trigger/api-gateway-proxy.d.ts#L65).